### ISSUE TITLE
DevPod starts with blank window and nothing works

### ISSUE THREAD

**Issue by todaywasawesome** [loft-sh/devpod#302]
**What happened?**
I installed the DevPod app from the website [https://devpod.sh/](https://devpod.sh/) (Apple Silicon)

![](https://uploads.linear.app/0aab256f-61c4-44f4-9aff-15fe09c2026b/56ebee84-316a-418b-a07f-d1b92a6598ca/1ee76336-94db-4171-afe8-696458a4a422)

**What did you expect to happen instead?**
The app to start.

**How can we reproduce the bug?** (as minimally and precisely as possible)
 🤷‍♀️

**Local Environment:**

* DevPod Version: 0.1.4 \[use `devpod --version`\]
* Operating System: mac
* ARCH of the OS: AMD64 | ARM64 | i386

**DevPod Provider:**

* Cloud Provider: not setup
* Kubernetes Provider: \[use `kubectl version`\]
* Local/remote provider: docker | ssh
* Custom provider: provide imported `provider.yaml` config file

**Anything else we need to know?**

Logs

```
default	10:00:08.852132-0600	DevPod	Received configuration update from daemon (initial)
default	10:00:08.852930-0600	DevPod	CHECKIN: pid=84938
default	10:00:08.869647-0600	DevPod	CHECKEDIN: pid=84938 asn=0x0-0x5e55e5 foreground=1
default	10:00:08.871766-0600	DevPod	FRONTLOGGING: version 1
default	10:00:08.871828-0600	DevPod	Registered, pid=84938 ASN=0x0,0x5e55e5
default	10:00:08.874679-0600	DevPod	BringForward: pid=84938 asn=0x0-0x5e55e5 bringForward=1 foreground=1 uiElement=0 launchedByLS=1 modifiersCount=1 allDisabled=0
default	10:00:08.874783-0600	DevPod	BringFrontModifier: pid=84938 asn=0x0-0x5e55e5 Modifier 0 hideAfter=0 hideOthers=0 dontMakeFrontmost=0 mouseDown=0/0 seed=0/0
default	10:00:08.874903-0600	DevPod	BringForward: pid=84938 asn=0x0-0x5e55e5
default	10:00:08.874933-0600	DevPod	SetFrontProcess: asn=0x0-0x5e55e5 options=0
default	10:00:08.880038-0600	DevPod	Current system appearance, (HLTB: 1), (SLS: 0)
default	10:00:08.880985-0600	DevPod	No persisted cache on this platform.
default	10:00:08.882098-0600	DevPod	Post-registration system appearance: (HLTB: 1)
default	10:00:08.914007-0600	DevPod	NSApp cache appearance:
-NSRequiresAquaSystemAppearance: 0
-appearance: (null)
-effectiveAppearance: <NSCompositeAppearance: 0x6000010c8c80
 (
    "<NSAquaAppearance: 0x6000010c8e80>",
    "<NSSystemAppearance: 0x6000010d1780>"
)>
default	10:00:08.967426-0600	DevPod	0x109038c40 - [PID=0] WebProcessCache::updateCapacity: Cache is disabled because process swap on navigation is disabled
default	10:00:08.970657-0600	DevPod	-[SOAuthorization init]  on <private>
default	10:00:08.970828-0600	DevPod	-[SOAuthorizationCore init]  on <private>
default	10:00:08.971060-0600	DevPod	<SOServiceConnection: 0x600003ed26c0>: new XPC connection
default	10:00:08.971148-0600	DevPod	0x109034a00 - [PID=0] WebProcessProxy::constructor:
default	10:00:08.971278-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=0] WebPageProxy::constructor:
default	10:00:08.971491-0600	DevPod	0x109034a00 - [PID=0] WebProcessProxy::addExistingWebPage: webPage=0x129044e18, pageProxyID=5, webPageID=6
default	10:00:08.971515-0600	DevPod	0x109038c40 - [PID=0] WebProcessCache::updateCapacity: Cache is disabled by client
default	10:00:08.972763-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=0] WebPageProxy::loadRequest:
default	10:00:08.972790-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=0] WebPageProxy::launchProcess:
default	10:00:08.972800-0600	DevPod	0x109034a00 - [PID=0] WebProcessProxy::removeWebPage: webPage=0x129044e18, pageProxyID=5, webPageID=6
default	10:00:08.973021-0600	DevPod	0x109034d00 - [PID=0] WebProcessProxy::constructor:
default	10:00:08.974970-0600	DevPod	0x109011c20 - [PID=0, throttler=0x109034e78] ProcessThrottler::Activity::Activity: Starting background activity / 'WebProcess initialization'
default	10:00:08.976299-0600	DevPod	<PKDiscoveryDriver:0x600001ad0420> created discovery driver
default	10:00:08.976336-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Beginning discovery for flags: 0, point: com.apple.services
default	10:00:08.976403-0600	DevPod	<PKDiscoveryDriver:0x600001acd860> created discovery driver
default	10:00:08.976435-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Beginning discovery for flags: 0, point: com.apple.ui-services
default	10:00:08.977468-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Completed discovery. Final # of matches: 0
default	10:00:08.977494-0600	DevPod	<PKDiscoveryDriver:0x600001ad0420> delivering update to host (0 plugins)
default	10:00:08.977596-0600	DevPod	<PKDiscoveryDriver:0x600001ad0420> installing watchers for continuous discovery
default	10:00:08.982161-0600	DevPod	<<<< Alt >>>> fpSupport_GetVideoRangeForCoreDisplayWithPreference: displayID 3 reported potentialHeadRoom=1 wideColorSupported=NO marz=NO almd=NO deviceAllowsHDR=YES isBuiltinPanel=NO externalPanel=YES prefersHDR10=NO
default	10:00:08.992259-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Completed discovery. Final # of matches: 2
default	10:00:08.992355-0600	DevPod	<PKDiscoveryDriver:0x600001acd860> delivering update to host (2 plugins)
default	10:00:08.996840-0600	DevPod	<PKDiscoveryDriver:0x600001acd860> installing watchers for continuous discovery
default	10:00:08.998337-0600	DevPod	<<<< Alt >>>> fpSupport_GetVideoRangeForCoreDisplayWithPreference: displayID 2 reported potentialHeadRoom=1 wideColorSupported=NO marz=NO almd=NO deviceAllowsHDR=YES isBuiltinPanel=NO externalPanel=YES prefersHDR10=NO
default	10:00:08.998785-0600	DevPod	0x109034a00 - [PID=0] WebProcessProxy::destructor:
default	10:00:08.998816-0600	DevPod	0x109034b78 - [PID=0] ProcessThrottler::invalidateAllActivities: BEGIN (foregroundActivityCount: 0, backgroundActivityCount: 0)
default	10:00:08.998827-0600	DevPod	0x109034b78 - [PID=0] ProcessThrottler::invalidateAllActivities: END
default	10:00:08.998838-0600	DevPod	0x109034d00 - [PID=0] WebProcessProxy::addExistingWebPage: webPage=0x129044e18, pageProxyID=5, webPageID=6
default	10:00:09.000911-0600	DevPod	[0x129045e20] CVCGDisplayLink::setCurrentDisplay: 0
default	10:00:09.001009-0600	DevPod	[0x129045e00] CVDisplayLinkCreateWithCGDisplays count: 1 [displayID[0]: 0x0] [CVCGDisplayLink: 0x129045e20]
default	10:00:09.001032-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=0] WebPageProxy::loadRequestWithNavigationShared:
default	10:00:09.001396-0600	DevPod	nw_path_evaluator_start [7D890091-2AC3-4B37-B664-75A911E2BCD7 <NULL> generic, attribution: developer]
	path: satisfied (Path is satisfied), interface: en0, ipv4, dns
default	10:00:09.002006-0600	DevPod	Faulting in CFHTTPCookieStorage singleton
default	10:00:09.002017-0600	DevPod	Creating default cookie storage with process/bundle identifier
default	10:00:09.002813-0600	DevPod	Faulting in NSHTTPCookieStorage singleton
default	10:00:09.003835-0600	DevPod	[0x13a826620] CVCGDisplayLink::setCurrentDisplay: 3
default	10:00:09.003860-0600	DevPod	[0x13a826600] CVDisplayLinkCreateWithCGDisplays count: 1 [displayID[0]: 0x3] [CVCGDisplayLink: 0x13a826620]
default	10:00:09.004352-0600	DevPod	SetFrontProcess: asn=0x0-0x5e55e5 options=1
default	10:00:09.005404-0600	DevPod	Registering for test daemon availability notify post.
default	10:00:09.005511-0600	DevPod	notify_get_state check indicated test daemon not ready.
default	10:00:09.005584-0600	DevPod	notify_get_state check indicated test daemon not ready.
default	10:00:09.040220-0600	DevPod	SignalReady: pid=84938 asn=0x0-0x5e55e5
default	10:00:09.041178-0600	DevPod	SIGNAL: pid=84938 asn=0x0x-0x5e55e5
default	10:00:09.058645-0600	DevPod	Initializing connection
default	10:00:09.058681-0600	DevPod	Removing all cached process handles
default	10:00:09.058726-0600	DevPod	Sending handshake request attempt #1 to server
default	10:00:09.058743-0600	DevPod	Creating connection to com.apple.runningboard
default	10:00:09.059586-0600	DevPod	Handshake succeeded
default	10:00:09.059600-0600	DevPod	Identity resolved as app<application.sh.loft.devpod.66206459.66206475(502)>
default	10:00:09.069626-0600	DevPod	0x109038c40 - [PID=0] WebProcessCache::setApplicationIsActive: (isActive=1)
default	10:00:09.093915-0600	DevPod	SetFrontProcess: asn=0x0-0x5e55e5 options=1
default	10:00:09.097908-0600	DevPod	filteredItemsFromItems:<private> [84938]--> <private>
default	10:00:09.097933-0600	DevPod	Requesting sharingServicesForFilteredItems:<private> mask:<private>
default	10:00:09.097950-0600	DevPod	Query extensions (async) for items: <private> onlyViewerOrEditor:1
default	10:00:09.098009-0600	DevPod	filteredItemsFromItems:<private> [84938]--> <private>
default	10:00:09.098026-0600	DevPod	Requesting sharingServicesForFilteredItems:<private> mask:<private>
default	10:00:09.098035-0600	DevPod	Query extensions (async) for items: <private> onlyViewerOrEditor:1
default	10:00:09.098179-0600	DevPod	0x109034d00 - [PID=84965] WebProcessProxy::didFinishLaunching:
default	10:00:09.101305-0600	DevPod	filteredItemsFromItems:<private> [84938]--> <private>
default	10:00:09.101322-0600	DevPod	Requesting sharingServicesForFilteredItems:<private> mask:<private>
default	10:00:09.101336-0600	DevPod	Query extensions (async) for items: <private> onlyViewerOrEditor:1
default	10:00:09.101388-0600	DevPod	Matching dictionary: <private>, attributesArray: <private>
default	10:00:09.101406-0600	DevPod	Discover extensions with attributes <private>
default	10:00:09.101391-0600	DevPod	Matching dictionary: <private>, attributesArray: <private>
default	10:00:09.101613-0600	DevPod	Discover extensions with attributes <private>
default	10:00:09.102190-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Beginning discovery for flags: 1024, point: com.apple.ui-services
default	10:00:09.102217-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Beginning discovery for flags: 1024, point: com.apple.ui-services
default	10:00:09.102324-0600	DevPod	Matching dictionary: <private>, attributesArray: <private>
default	10:00:09.102361-0600	DevPod	Discover extensions with attributes <private>
default	10:00:09.102422-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Beginning discovery for flags: 1024, point: com.apple.ui-services
default	10:00:09.108971-0600	DevPod	0x1090741a0 - NetworkProcessProxy is taking a background assertion because a web process is requesting a connection
default	10:00:09.116625-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Completed discovery. Final # of matches: 2
default	10:00:09.116790-0600	DevPod	2 plugins found
default	10:00:09.117600-0600	DevPod	Service with identifier <private> passes activation rule: 1
default	10:00:09.122828-0600	DevPod	UNIX error exception: 17
default	10:00:09.124845-0600	DevPod	Plugin <private> not enabled, skip it.
default	10:00:09.124881-0600	DevPod	1 compatible services found for attributes <private>
default	10:00:09.124938-0600	DevPod	Discovery done
default	10:00:09.124979-0600	DevPod	Discover extensions with attributes <private>
default	10:00:09.125148-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Beginning discovery for flags: 1024, point: com.apple.services
default	10:00:09.132025-0600	DevPod	UNIX error exception: 17
default	10:00:09.139817-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Completed discovery. Final # of matches: 2
default	10:00:09.140399-0600	DevPod	2 plugins found
default	10:00:09.140741-0600	DevPod	Service with identifier <private> passes activation rule: 0
default	10:00:09.140840-0600	DevPod	Service dictionary for plugin <private> not available, skip it.
default	10:00:09.141035-0600	DevPod	Plugin <private> not enabled, skip it.
default	10:00:09.141139-0600	DevPod	0 compatible services found for attributes <private>
default	10:00:09.141258-0600	DevPod	Discovery done
default	10:00:09.141371-0600	DevPod	Discover extensions with attributes <private>
default	10:00:09.141566-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Beginning discovery for flags: 1024, point: com.apple.services
default	10:00:09.142391-0600	DevPod	UNIX error exception: 17
default	10:00:09.153220-0600	DevPod	0x109011c20 - [PID=0, throttler=0x109034e78] ProcessThrottler::Activity::invalidate: Ending background activity / 'WebProcess initialization'
default	10:00:09.160839-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Completed discovery. Final # of matches: 2
default	10:00:09.161227-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Completed discovery. Final # of matches: 0
default	10:00:09.161319-0600	DevPod	2 plugins found
default	10:00:09.161387-0600	DevPod	0 plugins found
default	10:00:09.161475-0600	DevPod	0 compatible services found for attributes <private>
default	10:00:09.161591-0600	DevPod	Discovery done
default	10:00:09.161638-0600	DevPod	Service with identifier <private> passes activation rule: 0
default	10:00:09.161680-0600	DevPod	Completed querying extensions: <private>
default	10:00:09.161700-0600	DevPod	Service dictionary for plugin <private> not available, skip it.
default	10:00:09.161830-0600	DevPod	Plugin <private> not enabled, skip it.
default	10:00:09.161858-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Completed discovery. Final # of matches: 0
default	10:00:09.161879-0600	DevPod	0 compatible services found for attributes <private>
default	10:00:09.161973-0600	DevPod	0 plugins found
default	10:00:09.162074-0600	DevPod	Discovery done
default	10:00:09.162101-0600	DevPod	0 compatible services found for attributes <private>
default	10:00:09.162137-0600	DevPod	Discover extensions with attributes <private>
default	10:00:09.162153-0600	DevPod	Discovery done
default	10:00:09.162213-0600	DevPod	Completed querying extensions: <private>
default	10:00:09.162252-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Beginning discovery for flags: 1024, point: com.apple.services
default	10:00:09.162900-0600	DevPod	[d <private>] <PKHost:0x600002bc12c0> Completed discovery. Final # of matches: 0
default	10:00:09.162970-0600	DevPod	0 plugins found
default	10:00:09.163026-0600	DevPod	0 compatible services found for attributes <private>
default	10:00:09.163115-0600	DevPod	Discovery done
default	10:00:09.163182-0600	DevPod	Completed querying extensions: <private>
default	10:00:09.164932-0600	DevPod	Sorted services: <private>
default	10:00:09.165090-0600	DevPod	Sorted services: <private>
default	10:00:09.165179-0600	DevPod	Sorted services: <private>
default	10:00:09.175189-0600	DevPod	[0x129045e00] CVDisplayLinkStart
default	10:00:09.175223-0600	DevPod	[0x129045e20] CVDisplayLink::start
default	10:00:09.175282-0600	DevPod	[0x6000015ca0d0] CVXTime::reset
default	10:00:09.179903-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::decidePolicyForNavigationAction: frameID=4, navigationID=1
default	10:00:09.180051-0600	DevPod	0x1090118f0 - SOAuthorizationCoordinator::tryAuthorize
default	10:00:09.180798-0600	DevPod	-[SOConfigurationClient init]  on <private>
default	10:00:09.180876-0600	DevPod	<SOServiceConnection: 0x600003e88300>: new XPC connection
default	10:00:09.181039-0600	DevPod	0x1090118f0 - SOAuthorizationCoordinator::tryAuthorize: Cannot authorize the requested URL.
default	10:00:09.181304-0600	DevPod	[0x13a826600] CVDisplayLinkStart
default	10:00:09.181361-0600	DevPod	[0x13a826620] CVDisplayLink::start
default	10:00:09.181483-0600	DevPod	[0x6000015ca3e0] CVXTime::reset
default	10:00:09.181467-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::decidePolicyForNavigationAction: listener called: frameID=4, navigationID=1, policyAction=0, safeBrowsingWarning=0, isAppBoundDomain=0
default	10:00:09.181596-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::receivedNavigationPolicyDecision: frameID=4, navigationID=1, policyAction=0
default	10:00:09.181641-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::decidePolicyForNavigationAction: keep using process 84965 for navigation, reason=Process has not yet committed any provisional loads
default	10:00:09.185024-0600	DevPod	client.trigger:#N CCFG for cid 0x62 has # of profiles: 0
default	10:00:09.211201-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didStartProvisionalLoadForFrame: frameID=4
default	10:00:09.211221-0600	DevPod	0x109034d00 - [PID=84965] WebProcessProxy::didStartProvisionalLoadForMainFrame:
default	10:00:09.212255-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didNavigateWithNavigationDataShared:
default	10:00:09.212707-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didCommitLoadForFrame: frameID=4
default	10:00:09.223455-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didFinishLoadForFrame: frameID=4
default	10:00:09.278764-0600	DevPod	<<<< Alt >>>> fpSupport_GetVideoRangeForCoreDisplayWithPreference: displayID 3 reported potentialHeadRoom=1 wideColorSupported=NO marz=NO almd=NO deviceAllowsHDR=YES isBuiltinPanel=NO externalPanel=YES prefersHDR10=NO
default	10:00:09.278896-0600	DevPod	<<<< Alt >>>> fpSupport_GetVideoRangeForCoreDisplayWithPreference: displayID 2 reported potentialHeadRoom=1 wideColorSupported=NO marz=NO almd=NO deviceAllowsHDR=YES isBuiltinPanel=NO externalPanel=YES prefersHDR10=NO
default	10:00:09.279020-0600	DevPod	0x109098380 - GPUProcessProxy is taking a background assertion because a web process is requesting a connection
default	10:00:09.384742-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didNavigateWithNavigationDataShared:
default	10:00:09.384773-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didSameDocumentNavigationForFrame: frameID=4
default	10:00:09.389856-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didFinishDocumentLoadForFrame: frameID=4
default	10:00:09.440335-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didNavigateWithNavigationDataShared:
default	10:00:09.440390-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::didSameDocumentNavigationForFrame: frameID=4
default	10:00:09.464831-0600	DevPod	0x129044e18 - [pageProxyID=5, webPageID=6, PID=84965] WebPageProxy::updateActivityState: view visibility state changed 0 -> 1
default	10:00:09.712385-0600	DevPod	[0x129045e00] CVDisplayLinkStop
default	10:00:09.712446-0600	DevPod	[0x129045e20] CVDisplayLink::stop
default	10:00:09.845570-0600	DevPod	[0x13a826600] CVDisplayLinkStop
default	10:00:09.845679-0600	DevPod	[0x13a826620] CVDisplayLink::stop
default	10:00:09.968003-0600	DevPod	[0x13a826600] CVDisplayLinkStart
default	10:00:09.968030-0600	DevPod	[0x13a826620] CVDisplayLink::start
default	10:00:09.968099-0600	DevPod	[0x6000015ca3e0] CVXTime::reset
default	10:00:10.346743-0600	DevPod	[0x13a826600] CVDisplayLinkStop
default	10:00:10.346813-0600	DevPod	[0x13a826620] CVDisplayLink::stop
default	10:00:10.391412-0600	DevPod	[0x13a826600] CVDisplayLinkStart
default	10:00:10.391474-0600	DevPod	[0x13a826620] CVDisplayLink::start
default	10:00:10.391597-0600	DevPod	[0x6000015ca3e0] CVXTime::reset
default	10:00:10.779378-0600	DevPod	[0x13a826600] CVDisplayLinkStop
default	10:00:10.779468-0600	DevPod	[0x13a826620] CVDisplayLink::stop
default	10:00:11.234520-0600	DevPod	0x109038c40 - [PID=0] WebProcessCache::setApplicationIsActive: (isActive=0)
default	10:00:11.739481-0600	DevPod	[0x13a826600] CVDisplayLinkStart
default	10:00:11.739537-0600	DevPod	[0x13a826620] CVDisplayLink::start
default	10:00:11.740319-0600	DevPod	[0x6000015ca3e0] CVXTime::reset
```

**Comment by FabianKramm** [comment]
@todaywasawesome thanks for reporting this issue! We are currently investigating why this can happen on some Macs, we found the issue for some linux distributions, but for mac os we couldn't reproduce so far. What MacOS version do you have? You stated its apple silicon, is it M1 or M2 generation?

**Comment by FabianKramm** [comment]
@todaywasawesome yeah since its a prerelease you need to download it from the Releases page

**Comment by todaywasawesome** [comment]
@FabianKramm same result. 
![Screen Shot 2023-05-17 at 11 15 45 AM](https://github.com/loft-sh/devpod/assets/3849900/90a2103f-0f5d-414d-b196-6a31ec6db88b)

**Comment by todaywasawesome** [comment]
Could be because I'm still on MacOS Monterey 12.2. I'm upgrading now. I'll let you know if it works in 13.3.1.

**Comment by oopsforgot** [comment]
Same issue for me on an M1 MacOS Monterey 12.2.1 and Devpod 0.1.5.

For what it's worth the Devpod CLI works without issue.

**Comment by todaywasawesome** [comment]
@FabianKramm Ok after upgrading to macOS 13.3.1 it's working. It appears to be incompatible with Monterey.

**Comment by tylerauerbeck** [comment]
Just another +1 on this not working on older versions of macOS. Works fine on 13.X machine, just gives the empty window on versions prior to that.

**Comment by pascalbreuninger** [comment]
Thanks for reporting, we'll take a look at the older Mac versions

**Comment by pascalbreuninger** [comment]
Issue should be resolved when v0.1.5 goes live
